src/common/modules/navigations/gnb/modules/gnb-menu/GNBMenu.vue

```diff
@@ -100,16 +96,22 @@ export default defineComponent({
         },
     },
     setup(props, { emit }) {
-        const openMenu = () => {
-            emit('open-menu', props.name);
+        const handleMenu = () => {
+            if (props.subMenuList?.length > 0) {
+                emit('open-menu', props.name);
+            } else {
+                const is_duplicate = SpaceRouter.router.currentRoute.name === props.name;
```

Comment:
```suggestion
                const isDuplicate = SpaceRouter.router.currentRoute.name === props.name;
```
프론트에서 사용하는 변수명으로는 카멜케이스로 작성 부탁드려요!

Reply:
넵!!